.github/actions/rust-lints/action.yaml

```diff
@@ -17,5 +17,5 @@ runs:
     - name: Run cargo sort and rust lint checks
       shell: bash
       run: |
-        cargo install cargo-sort
+        cargo install cargo-sort --version 1.0.7  # Fix the version to avoid unexpected changes
```

Comment:
Is it not possible to use a lockfile for this ?

Reply:
There is no `Cargo.toml` for `cargo-sort` AFAICT. It's an indirect dependency outside the workspace.